lib/types.ts

```diff
@@ -33,20 +38,12 @@ export type FactType =
   | "dockerfileAnalysis"
   | "rootFs"
   | "imageId"
-  | "imageOsReleasePrettyName";
+  | "imageOsReleasePrettyName"
+  | "manifestFiles";
```

Comment:
do we want to change to a more container specific name? just in case someone else in the future will want to add manifest file as a fact